api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirTaskDaoImpl.java

```diff
@@ -45,4 +48,10 @@ public Task saveTask(Task task) throws DAOException {
 	public Task getTaskByUuid(String uuid) {
 		return (Task) sessionFactory.getCurrentSession().createCriteria(Task.class).add(eq("uuid", uuid)).uniqueResult();
 	}
+
+	@Override
+	public Collection<Task> getTasksByBasedOnUUID(String resourceType, String orderUuid) {
```

Comment:
1. This should probably be `getTasksByBasedOnUuid()` for consistency
2. `orderUuid` should probably just be `uuid`.